contracts/p1/BasketHandler.sol

```diff
@@ -764,5 +808,5 @@ contract BasketHandlerP1 is ComponentP1, IBasketHandler {
      * variables without shifting down storage in the inheritance chain.
      * See https://docs.openzeppelin.com/contracts/4.x/upgradeable#storage_gaps
      */
-    uint256[42] private __gap;
+    uint256[41] private __gap;
```

Comment:
Did you check this? The 2 uint48s and the CollateralStatus enum all fit in one slot?

Reply:
Yep.. totally checked

Reply:
Yes, checked.  Its packing everything in one slot because we only have few values in that enum. We have room for 160 values we could store in that slot so I think we are ok.

Reply:
But it is true enums could take up to a full slot, its not the case for this enum so I think we are safe.

Reply:
Sweet. I'm gonna end up packing another `uint48` in that storage slot in #771